flask-server/main.py

```diff
@@ -377,16 +377,15 @@ def fetch_write_clinks():
 @app.route('/apis/fetch-bookmarks', methods=['GET'])
```

Comment:
What I meant before is to use resourceful url like `/apis/fetch-bookmarks/<string:clink_id>`
and you can use the clink_id as `def fetch_bookmarks(clink_id)`
Read more: https://flask-restful.readthedocs.io/en/latest/quickstart.html#resourceful-routing